app/components/visualizer/main-viz/main-viz.jsx

```diff
@@ -1,3 +1,26 @@
 import DonutChartComponent from "./templates/DonutChart/DonutChartComponent.jsx";
 
-React.render(<DonutChartComponent />, document.getElementById("main-viz"));
+class MainViz extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
`super(props)` calls the parent constructor (`React.Component`) function. This initializes the component.